cognite/powerops/resync/models/v1/config_to_model/to_production_model.py

```diff
@@ -159,16 +159,23 @@ def to_production_model(configuration: config.ProductionConfig) -> ProductionMod
             else:
                 mappings = {}
 
+            all_connections = shop_case["connections"]
+            all_junctions = shop_case["model"].get("junction")
+            all_tunnels = shop_case["model"].get("tunnel")
+            inlet_reservoir_name, connection_losses = _plant_to_inlet_reservoir_with_losses(
+                plant_name, all_connections, all_junctions, all_tunnels, {r.name for r in model.reservoirs}
+            )
+            sum_plant_losses = float(attributes.get("main_loss", [head_loss_factor_fallback])[0]) + connection_losses
+
             plant = Plant(
                 name=plant_name,
                 display_name=display_name,
                 ordering=str(order),
                 outlet_level=float(attributes.get("outlet_line", 0)),
                 p_min=float(attributes.get("p_min", p_min_fallback)),
                 p_max=float(attributes.get("p_max", p_max_fallback)),
-                head_loss_factor=float(
-                    attributes.get("main_loss", [head_loss_factor_fallback])[0]
-                ),  # For some reason, SHOP defines this as a list, but we only need the first (and only) value
+                head_loss_factor=sum_plant_losses,
```

Comment:
Okay, I think I see the reason the head_loss_factor changes now. But is this a good idea? i.e., to add the connection losses to the main head loss factor **and** add the connection losses as a separate loss? I would strongly suggest not to to it.

Reply:
The reason for adding connection_losses as a metadata field as well was by request from Øicind cause he wanted a way of distinguishing what plants had connection losses and what plants did not. And I thought we agreed from the meeting with them that we should add the losses to the head_loss_factor? Or do you suggest not to sum them and have the function/consumer sum the two losses so that it is not "implicit" that head_loss_factors contains the sum

Reply:
Yes. I would avoid having it implicit. 

I agreed the original plan was to add it direct to the "main head loss", but that was also when we did not plan to add the connection losses metadata field. At least that's what I thought?

Reply:
Ok, should I remove the summation then? and then we do the sum in the function later on?